internal/build/docker.go

```diff
@@ -102,6 +103,21 @@ func (l *localDockerBuilder) buildDocker(ctx context.Context, df Dockerfile,
 	return resultDigest, nil
 }
 
+// Tag the digest with the given name and wm-tilt tag.
+func (l *localDockerBuilder) TagDocker(ctx context.Context, ref reference.Named, dig digest.Digest) (reference.NamedTagged, error) {
+	tag, err := reference.WithTag(ref, tiltTag)
+	if err != nil {
+		return nil, fmt.Errorf("TagDocker: %v", err)
+	}
+
+	err = l.dcli.ImageTag(ctx, dig.String(), tag.String())
```

Comment:
should we put some spans around this?

Reply:
i think a good rule of thumb is to only add spans where we have good reason to think we need them; the bigger risk is wasting a lot of engineering time creating spans we never use